intro-to-tidyverse.rmd

```diff
@@ -972,16 +994,20 @@ untidy_df %>%
 
 ```{r, highlight.output = c(2, 4, 9), eval = FALSE}
 untidy_df %>% 
-  gather(gender_year, income, male_2016:female_2018) %>% 
-  separate(gender_year, into = c("gender", "year")) #<<
+  pivot_longer(cols = male_2016:female_2018, 
+               names_to = c("gender", "year"),
+               names_sep = "_", #<<
```

Comment:
this is an interesting new addition that seems like it bypasses `separate()` definitely useful to show both.